TCA Counter App/EditCount/CountFeature.swift

```diff
@@ -33,24 +35,24 @@ enum CounterAction {
 }
 
 struct CounterEnvironment {
-    static let Max = 5
+    static let Max = 9
```

Comment:
Instead of defining these as `static let`, let's also define them as functions. This will allow us to provide stub values for max and min during unit tests.